### PR TITLE
Create Get, Set, Remove-SPOContentEventEmailAddresses.md

### PR DESCRIPTION
**Pull request by Mohammed-Shabaz**
Adding the cmdlet Remove-SPOContentEventEmailAddresses.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentEventEmailAddresses.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Remove-SPOContentEventEmailAddresses
+
+## SYNOPSIS
+
+Removes the email addresses associated with the specified category of content event if they exist. Consequently, notification emails will no longer be sent to these addresses.
+
+## SYNTAX
+
+```powershell
+Remove-SPOContentEventEmailAddresses [-Category] <ContentEventCategory> [-EmailAddresses] <List of comma seperated email addresses>
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentEventEmailAddresses.md:23]: ><List of comma seperated email addresses> [](http://example.com/codeflow?start=90&length=41)

`<string[]>`

**Comment by samkabue** [comment]
> 

link  to get-/set- cmdlets. Search other files for how they do it

---
Refers to: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentEventEmailAddresses.md:77 in a31b556. [](commit_id = a31b556fb6b414039890849157f5df71755a0fb8, deletion_comment = False)

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentEventEmailAddresses.md (hunk 1 above)
Comment (Mohammed-Shabaz) [reply sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentEventEmailAddresses.md]: Done

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentEventEmailAddresses.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Get-SPOContentEventEmailAddresses -Category Ransomware
+```
+
+In Example 1, the system will display the email addresses categorized under ransomware that the admin had previously entered.
+
+## PARAMETERS
+
+### -Category
+
+Specifies the Content event category.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentEventEmailAddresses.md:44]: >Content event category [](http://example.com/codeflow?start=14&length=22)

what's the right casing? e.g., as compared to lines 18 or 28?

Comment (samkabue) [reply sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentEventEmailAddresses.md:44]: same to the other files

Comment (Mohammed-Shabaz) [reply sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentEventEmailAddresses.md]: done

**Review by samkabue — APPROVED** [review]
:shipit:

**Comment by Mohammed-Shabaz** [comment]
Hi team,

This is merged.

Thanks,
Regards,
Mohammed Shabaz
________________________________

**Pull request merged**